Cirrious/Cirrious.MvvmCross.Droid.FullFragging/Fragments/MvxDialogFragment.cs

```diff
@@ -0,0 +1,69 @@
+// MvxDialogFragment.cs
+// (c) Copyright Cirrious Ltd. http://www.cirrious.com
+// MvvmCross is licensed using Microsoft Public License (Ms-PL)
+// Contributions and inspirations noted in readme.md and license.txt
+// 
+// Project Lead - Stuart Lodge, @slodge, me@slodge.com
+
+using Android.OS;
+using Android.Runtime;
+using Cirrious.MvvmCross.Binding.BindingContext;
+using Cirrious.MvvmCross.Droid.FullFragging.Fragments.EventSource;
+using Cirrious.MvvmCross.ViewModels;
+using System;
+
+namespace Cirrious.MvvmCross.Droid.FullFragging.Fragments
+{
+    public abstract class MvxDialogFragment
+        : MvxEventSourceDialogFragment
+          , IMvxFragmentView
+    {
+        protected MvxDialogFragment(IntPtr javaReference, JniHandleOwnership transfer)
+            : base(javaReference, transfer)
+        {
+            this.AddEventListeners();
+        }
+
+        protected MvxDialogFragment()
+        {
+            this.AddEventListeners();
+        }
```

Comment:
Doesn't this mean that `AddEventListeners()` could happen twice?

See http://developer.xamarin.com/guides/android/under_the_hood/architecture/#Java_Activation